package/src/components/CartCheckoutButton/v1/CartCheckoutButton.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
Since this component has a click handler I'd write a test for that using enzyme. 
Something like [this](https://gist.github.com/nnnnat/fc035b7f96b39b2b099fc178ee529fb7) should cover it